### ISSUE TITLE
Home -> Header Missing Accessible Indication of Toggle Control State

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/MathShare#758]
The “Help Center” button acts as a disclosure control.  That is, it can be used to toggle the display of the Help Center content.  On initial page load, the Help Center content is hidden, but the required aria-expanded attribute is not present on this button to indicate this fact (or that the control will act as a toggle at all). 
 
An aria-expanded attribute is added to the control once it has been activated for the first time, and from this point onward the value of the attribute is correctly updated in line with the control’s state.  However, it must be in place from the very first time that the control is rendered. 

Suggested Solution 
 
Ensure that this (and all other toggle/disclosure controls) have an aria-expanded attribute applied as soon as they are initially rendered.  Here, the initial value should be set to “false”. 
 
Proposed Code Sample 
 
File: src/components/Home/components/Header/index.js 
 
`<button className={`nav-link dropdown-toggle btn ${header.dropDownMenu}`} id={questionBtnId} … aria-expanded="false" …> `

**Comment by jscholes** [comment]
Fixed in https://github.com/benetech/MathShare/commit/a658015b7f1ce8c6fdbe279aee56a2a6bcd71b6f.
